frontend/src/metabase/lib/card.js

```diff
@@ -26,17 +26,38 @@ export function startNewCard(type, databaseId, tableId) {
 // TODO: move to redux
 export async function loadCard(cardId, { dispatch, getState }) {
   try {
-    await dispatch(Questions.actions.fetch({ id: cardId }, { reload: true }));
```

Comment:
@deniskaber you should be able to pass required `properties` here. If they aren't present on the object that is found by ID, it would be re-fetched. Like here https://github.com/metabase/metabase/blob/1efa6c6f000cb4bd38065563176c35ea0027c865/frontend/src/metabase/nav/containers/CollectionBreadcrumbs/CollectionBreadcrumbs.tsx#L9